bin/node/runtime/src/lib.rs

```diff
@@ -931,7 +931,7 @@ construct_runtime!(
 );
 
 /// The address format for describing accounts.
-pub type Address = <Indices as StaticLookup>::Source;
+pub type Address = sp_runtime::MultiAddress<AccountId, AccountIndex>;
```

Comment:
Why is this change? The old code has less duplication. Or did it stop working for some reason?

Reply:
I think it was just me being explicit. If there is no different, i would personally prefer explicitness. Else, if there is good reason to keep it the same, i dont really care about this change.

Reply:
Personally, I would always prefer writing things as generic as possible and thus reducing code churn when a refactoring is made. Ultimately it is your decision. I just pointed it out here because I thought you made this change by mistake.